tests/unit/conftest.py

```diff
@@ -1297,3 +1323,11 @@ def mock_datacite_minting(mocker, app):
         'invenio_pidstore.providers.datacite.DataCiteMDSClient')
     yield datacite_mock
     app.config['DEPOSIT_DATACITE_MINTING_ENABLED'] = orig
+
+
+@pytest.fixture
+def use_flask_limiter(app):
```

Comment:
I would move this next to the `wrap_rate_limit` function, just to know that these two go together.